packages/toolkit/assistive/form-field-error.css

```
@@ -1,97 +1,168 @@
+  --_error-clr-danger: #db1818;
+  --_error-clr-warning: #a16207;
+  --_error-clr-danger-dark: #fca5a5;
+  --_error-clr-warning-dark: #fcd34d;
+
+  --_error-token-transparent: transparent;
+  --_error-token-disc: disc;
+  --_error-token-outside: outside;
+  --_error-border-width-default: 0;
+  --_error-border-radius-default: 0;
+  --_error-border-style-default: solid;
+  --_error-animation-default: ngxStatusSlideIn 300ms
```

Stylelint (17.7.0)

[error] Expected "ngxStatusSlideIn" to be "ngxstatusslidein" (value-keyword-case)

(value-keyword-case)